doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
Eventually, a json schema might be useful for validation.

Reply:
I agree. But only when we have agreed on a structure.